### ISSUE TITLE
Clarification on table 5

### ISSUE THREAD

**Issue by ealmazanm** [clovaai/bros#17]
Hi there, first of all thanks for sharing your excellent work.
I have a doubt regarding how you get the results of table 5. In the paper you mention that you don't use the order information, but how do you implement that exactly?. Do you remove the 1D abs. positional embeddings from the model? if so, that comes with a new pre-training? and finally, I guess you still train with the dataset order of the words and it is only in test where you shuffle the words, is that right?

Thanks in advance!
